versions.yaml

```diff
@@ -108,6 +108,7 @@ module-sets:
       - github.com/open-telemetry/opentelemetry-collector-contrib/internal/datadog
       - github.com/open-telemetry/opentelemetry-collector-contrib/pkg/batchperresourceattr
       - github.com/open-telemetry/opentelemetry-collector-contrib/pkg/batchpersignal
+      - github.com/open-telemetry/opentelemetry-collector-contrib/pkg/codec
```

Comment:
This is not needed, right?

Reply:
ah, nope. Too many files flying around... I'll fix it.